docs/source/getting_started/index.ipynb

```diff
@@ -1164,9 +1164,9 @@
    "id": "762799a5-5b99-46ca-8b68-401dfdf10100",
    "metadata": {},
    "source": [
-    "To include multiple years of data, you can include the years in the \"year\" parameter in the form of [Start Year, Stop Year]\n",
+    "To request multiple years of data, you can include the years in the \"year\" parameter in the form of `[Start Year, Stop Year]`.\n",
     "\n",
-    "This also works in a dataset with only a single year of data"
+    "Date ranges can only be used for multi-year datasets"
```

Comment:
I reworded the statement with your suggestion to highlight that this parameter can only be used for multi-year data

Reply:
This is good. I like the rewritten 2nd line. 

One more thing, can you add something like " (i.e. Year Column is "MULTIPLE" in the `src.datasets` table or equivalently, coverage_start to coverage_end columns span multiple years)" at the end of the 2nd line"? 

That isn't needed specifically for your issue, but I think it would generally be good to define what a multi-year dataset is because it is not currently done anywhere else.

After that, I'll merge the pull request.